rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -385,40 +407,12 @@ def _set_features(
         attribute: Text,
     ) -> None:
         for index, example in enumerate(examples):
-            _sequence_features = Features(
-                sequence_features[index],
-                FEATURE_TYPE_SEQUENCE,
-                attribute,
-                self.component_config[FEATURIZER_CLASS_ALIAS],
-            )
-            example.add_features(_sequence_features)
-
-            _sentence_features = Features(
-                sentence_features[index],
-                FEATURE_TYPE_SENTENCE,
-                attribute,
-                self.component_config[FEATURIZER_CLASS_ALIAS],
+            self.add_features_to_message(
```

Comment:
such a good idea to pull this method out 💯

Reply:
:D yeah I also enjoy seeing this